testing/config/resources.json

```diff
@@ -0,0 +1,557 @@
+      },
+      {
+        "1": [
+          "failing"
+        ]
+      }
+    ],
+    "stdin": false,
+    "subdue": null,
+    "cron": "",
+    "ttl": 0,
+    "timeout": 0,
+    "round_robin": false,
+    "output_metric_format": "",
+    "output_metric_handlers": []
+  }
+}
+{
+  "type": "CheckConfig",
+  "spec": {
+    "command": "echo passing round robin && exit 0",
+    "environment": "default",
+    "handlers": [
+      "filtered-cat"
+    ],
+    "high_flap_threshold": 0,
+    "interval": 60,
+    "low_flap_threshold": 0,
+    "name": "passing-round-robin",
+    "organization": "default",
+    "publish": true,
+    "runtime_assets": [],
+    "subscriptions": [
+      "schedules"
+    ],
+    "proxy_entity_id": "",
+    "check_hooks": [
+      {
+        "0": [
+          "passing"
+        ]
+      },
+      {
+        "1": [
+          "failing"
+        ]
+      }
+    ],
+    "stdin": false,
+    "subdue": null,
+    "cron": "",
+    "ttl": 0,
+    "timeout": 0,
+    "round_robin": false,
+    "output_metric_format": "",
+    "output_metric_handlers": []
+  }
+}
+{
+  "type": "CheckConfig",
+  "spec": {
+    "command": "echo {{ .ID }}",
+    "environment": "default",
+    "handlers": [],
+    "high_flap_threshold": 0,
+    "interval": 10,
+    "low_flap_threshold": 0,
+    "name": "token-substitution",
+    "organization": "default",
+    "publish": true,
+    "runtime_assets": [],
+    "subscriptions": [
+      "misc",
+      "rbac"
+    ],
+    "proxy_entity_id": "",
+    "check_hooks": null,
+    "stdin": false,
+    "subdue": null,
+    "ttl": 0,
+    "timeout": 0,
+    "round_robin": false,
+    "output_metric_format": "",
+    "output_metric_handlers": []
+  }
+}
+{
+  "type": "HookConfig",
+  "spec": {
+    "name": "passing",
+    "command": "echo passing",
+    "timeout": 60,
+    "stdin": false,
+    "environment": "default",
+    "organization": "default"
+  }
+}
+{
+  "type": "HookConfig",
+  "spec": {
+    "name": "failing",
+    "command": "echo failing",
+    "timeout": 60,
+    "stdin": false,
+    "environment": "default",
+    "organization": "default"
+  }
+}
+{
+  "type": "Handler",
+  "spec": {
+    "name": "cat",
+    "type": "pipe",
+    "command": "cat",
+    "timeout": 0,
+    "handlers": [],
+    "filters": [],
+    "env_vars": [],
+    "environment": "default",
+    "organization": "default"
+  }
+}
+{
+  "type": "Handler",
+  "spec": {
+    "name": "filtered-cat",
+    "type": "pipe",
+    "command": "cat",
+    "timeout": 0,
+    "handlers": [],
+    "filters": [
+      "handle-failing-only"
+    ],
+    "env_vars": [],
+    "environment": "default",
+    "organization": "default"
+  }
+}
+{
+  "type": "EventFilter",
+  "spec": {
+    "name": "handle-failing-only",
+    "action": "allow",
+    "statements": [
+      "event.Check.Status == 1"
+    ],
+    "environment": "default",
+    "organization": "default"
+  }
+}
+{
+  "type": "Asset",
+  "spec": {
+    "name": "foo",
+    "url": "http:foo.com",
```

Comment:
URL is malformed

Reply:
@echlebek I'll get that changed 🤦‍♀️ 
https://github.com/sensu/sensu-go/blob/3c7b250ab400eac3cca06d136564542835e883de/testing/e2e/asset_store_test.go#L29-L34

![giphy](https://user-images.githubusercontent.com/10634078/41478365-44ad3804-707c-11e8-8432-938dff026d6f.gif)